parser.go

```diff
@@ -358,7 +358,7 @@ func (r *Rule) option(key item, l *lexer) error {
 		}
 		v.NumBytes = n
 
-		offset, err := strconv.Atoi(parts[1])
+		offset, err := strconv.Atoi(strings.TrimSpace(parts[1]))
```

Comment:
This seems to be unrelated to the change to add flowbits? Probably put this in its own PR

Reply:
> Do you have a specific need for flowbits to be their own struct? If not it may suffice simply to add "flowbits" to the keywords allowed for tags here:
> https://github.com/google/gonids/blob/master/parser.go#L178

flowbits maybe occur more than once,so  deal with it addtionally is better. also additional processing of byte_test keyword might be better,the rule like below contains four byte_test:
**alert tcp any any -> $HOME_NET !$HTTP_PORTS (msg:"ET CURRENT_EVENTS Malformed HeartBeat Request"; flow:established,to_server; content:"|18 03|"; depth:2; byte_test:1,<,4,2; content:"|01|"; offset:5; depth:1; byte_extract:2,3,record_len; byte_test:2,>,2,3; byte_test:2,>,record_len,6; threshold:type limit,track by_src,count 1,seconds 120; flowbits:set,ET.MalformedTLSHB; reference:cve,2014-0160; reference:url,blog.inliniac.net/2014/04/08/detecting-openssl-heartbleed-with-suricata/; reference:url,heartbleed.com/; reference:url,blog.fox-it.com/2014/04/08/openssl-heartbleed-bug-live-blog/; classtype:bad-unknown; sid:2018372; rev:2; metadata:created_at 2014_04_08, updated_at 2014_04_08;)**

**alert ip $HOME_NET any -> [184.106.133.130,184.73.167.34] any (msg:"CNC Shadowserver Reported CnC Server IP group 10"; reference:url,doc.emergingthreats.net/bin/view/Main/BotCC; reference:url,www.shadowserver.org; threshold: type limit, track by_src, seconds 3600, count 1; flowbits:set,ET.Evil; flowbits:set,ET.BotccIP; classtype:trojan-activity; sid:2404009; rev:5417; metadata:affected_product Any, attack_target Any, deployment Perimeter, tag Shadowserver, signature_severity Major, created_at 2012_05_04, updated_at 2019_07_03;)**

Reply:
> This seems to be unrelated to the change to add flowbits? Probably put this in its own PR

sorry for that, I forgot delete this test code, I will delete it.